Makefile

```diff
@@ -112,6 +112,18 @@ pause-container-release: pause-container
 	mkdir -p "$(shell pwd)/out"
 	@docker save ${PAUSE_CONTAINER_IMAGE}:${PAUSE_CONTAINER_TAG} > "$(shell pwd)/out/${PAUSE_CONTAINER_TARBALL}"
 
+# Variable to determine branch/tag of amazon-ecs-cni-plugins
+revision=master
```

Comment:
1. We could move it to be the top. 
2. I like this name `ECS_CNI_REPOSITORY_REVISION` but isn't it a bit too long ?

Also I was thinking if we should make this a target specific variable.